.gitignore

```diff
@@ -35,3 +36,5 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+static
```

Comment:
I think we're going to need this because we're using the repo to deploy to Heroku.